Cargo.toml

```diff
@@ -55,3 +55,15 @@ default-members = [
 ring = { git = "https://github.com/crypto-com/ring.git", rev = "bdbcc7041095f028d49d9fecd7edcf26d6083274" }
 # FIXME: use upstream when merged
 sha2 = { git = "https://github.com/crypto-com/hashes.git", rev = "289d5b76f2163a3808010341ed1df3cb156d97e1" }
+
+[patch.crates-io.tendermint]
```

Comment:
any PRs waiting for upstream?

Reply:
the pr is merged now, the patch is changed too.